app/Models/Cpt_Example.php

```diff
@@ -11,7 +11,7 @@
  */
 class Cpt_Example extends Model {
```

Comment:
you should create model to load members.

Reply:
done